assert.ts

```diff
@@ -0,0 +1,112 @@
+interface Assert<T> {
```

Comment:
This is completely unrelated to this PR, I suggest to revert it

Reply:
Okay, I will revert this change